streetcrm/static/js/inline_ajax.js

```diff
@@ -215,25 +215,10 @@ function getFieldsConfig() {
     return getInlineConfig()["fields"];
 }
 
-/* Isn't it rare and beautiful to use a switch statement with
```

Comment:
This reminds me of [Duff's Device](https://en.wikipedia.org/wiki/Duff%27s_device), but still, I'm glad it's out of our code now :-).